aiida/orm/comments.py

```diff
@@ -34,6 +34,23 @@ def delete(self, comment_id):
             """
             self._backend.comments.delete(comment_id)
 
+        def delete_all(self):
+            """
+            Remove all Comments from the collection
+            """
+            self._backend.comments.delete_all()
+
+        def delete_many(self, filters):
+            """
+            Delete Comments based on `filters`.
+            If `filters` is an empty dict, raise ValidationError.
+            If at least one entity is found, that entity is deleted, nothing is raised.
+
+            :param filters: similar to QueryBuilder filters
+            :type filters: dict
```

Comment:
add the note of exception being raised as `:raises ValidationError: if filters is an empty dictionary`